mist-lib/src/services/SignalTransductionService.js

```diff
@@ -0,0 +1,46 @@
+'use strict'
+
+const csv = require('csvtojson'),
+	path = require('path')
+
+const DefaultTableFilePath = path.resolve(__dirname, '..', 'signal-transduction', 'mist3-sig-trans-2017.csv')
+
+
+module.exports =
+class SignalTransductionService {
+	constructor(tableFilePath) {
+		this.tableFilePath = tableFilePath ? tableFilePath : DefaultTableFilePath
+		this.pfql = []
+		this.sigData = []
+	}
+
+	parseTable_() {
```

Comment:
Move parsing and generation of pfql rules into separate utility function (perhaps inside a `stp-utils.js` file?).

Reply:
Done